### DISCUSSION TITLE
Add a new project to leapcode

### DISCUSSION THREAD

**Discussion by stanly-johnson** [Leapcode-Open/leapcode-frontend#17] [General]
This space is meant to have a conversation about adding your project to leapcode. We plan to create a proper pipeline for this soon. but for now this process is manual. We are open to accepting any projects, although we prioritise projects with these qualities

1. The major language of the repo must be either Javascript, Python, Java or Shellscript
2. Anyone should be able to open PR (no requirement of joining another channel/group)
3. Actively Maintained (At least one beginner friendly issue in last 30 days)

**Comment by adityabisoi** [comment]
Hi,
I want to add my project to the list. The project is called `REST-fetch`. It is a project which has a greater aim of providing free endpoints to developers and testers for availing demo placeholder data in the form of JSON. Students will be able to learn JavaScript, TypeScript, NodeJS, ReactJS, and REST APIs on contributing to the project.
Here is the GitHub link: https://github.com/adityabisoi/REST-fetch
The project is actively maintained and has beginner as well as expert-level issues.
